packages/analytics/matomo/usePaq.ts

```diff
@@ -1,6 +1,14 @@
+declare global {
```

Comment:
Forgot to delete this file.

Reply:
https://github.com/iFixit/react-commerce/pull/1447/commits/580e167ca687a70c98ee0b35b8c3688cf6c78177